src/components/BorrowerProfile/JournalUpdates.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+	<section>
+		<!-- Display Content -->
+		<h2 class="tw-text-h2 tw-mb-3" :data-testid="`bp-updates-header`">
+			<!-- eslint-disable-next-line max-len -->
+			{{ sectionTitle }} <kv-loading-placeholder v-if="firstLoading" class="tw-inline-block tw-align-middle tw-mb-0.5" :style="{width: '27%', height: '28px' }" />
+		</h2>
+
+		<div
+			:data-testid="`bp-updates-section`"
+		>
+			<update-details
+				v-for="(journal, index) in journals" :key="journal.id"
+				:body="journal.body"
+				:date="journal.date"
+				:subject="journal.subject"
+				:index="totalItemCount - index"
+				:image-url="journal.image ? journal.image.url : null"
+			/>
+		</div>
+
+		<div v-if="loading" class="tw-rounded tw-bg-primary tw-p-3 tw-mb-3">
+			<!-- Loading placeholder for journal elements -->
+			<kv-loading-placeholder
+				class="tw-rounded tw-mb-2 tw-w-full" :style="{ height: '121px' }"
+			/>
+			<kv-loading-placeholder
+				class="tw-mb-2" :style="{ height: '14px', width: '96%' }"
+			/>
+			<kv-loading-placeholder
+				class="tw-mb-2" :style="{ height: '14px', width: '41%' }"
+			/>
+			<div class="tw-flex tw-justify-between tw-align-baseline">
+				<kv-loading-placeholder class="tw-mb-2" :style="{height: '28px', width: '33%'}" />
+				<kv-loading-placeholder class="tw-mb-2" :style="{height: '14px', width: '24%'}" />
+			</div>
+		</div>
+
+		<!-- Load More Button -->
+		<div v-if="journals.length < totalItemCount" class="tw-mb-3 tw-text-center">
+			<kv-button
+				class="tw-mb-3"
+				@click="loadMore"
+				variant="secondary"
+				:disabled="loading"
+				:state="buttonState"
+				:aria-label="`See older updates`"
+			>
+				See older updates
+			</kv-button>
+		</div>
+	</section>
+</template>
+
+<script>
+import { gql } from '@apollo/client';
+import { createIntersectionObserver } from '@/util/observerUtils';
+// TODO: replace the loading placeholder with component from kv-components when available.
```

Comment:
The loader is in kv-components.

Reply:
Nice, that was just some copy pasta :) I can open a PR to replace all these